syntax = "proto3";
message gameState{
    message playerState {
            int32 hp = 1;
            int32 bullets = 2;
            int32 grenades = 3;
            float shield_time = 4;
            int32 shield_health = 5;
            int32 num_deaths = 6;
            int32 num_shield = 7;
            string action = 8;
        }
    playerState p1 = 1;
    playerState p2 = 2;
};
